docs/reference/workflow-reference.md

```diff
@@ -1,3 +1,491 @@
+| `scene_ids`          | **list[str]**<br/>The scene IDs. If used, all other parameters except `geometry` are ignored.                              |
+| `asset_ids`          | **list[str]**<br/>The asset IDs. Use with Processing from Storage block.                                                   |
+
+<h5> Example </h5>
+
+```python
+workflow.construct_parameters(
+    geometry=up42.get_example_aoi(location="Berlin"),
+    geometry_operation="bbox",
+    start_date="2020-01-01",
+    end_date="2022-12-31",
+    limit=1,
+)
+```
+
+### construct_parameters_parallel()
+
+The `construct_parameters()` function allows you to fill out the JSON parameters for multiple jobs to run in parallel.
+
+```python
+construct_parameters_parallel(
+        geometries,
+        interval_dates,
+        scene_ids,
+        limit_per_job,
+        geometry_operation,
+)
+```
+
+The returned format is `list[dict]`.
+
+<h5> Arguments </h5>
+
+| Argument             | Overview                                                                                                                                                     |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `geometries`         | **list[Union[dict, Feature, geojson_Polygon, Polygon]]**<br/>The geometries of interest.                                                                     |
+| `interval_dates`     | **list[tuple[str, str]]**<br/>The start and end dates in the `YYYY-MM-DD` format.                                                                            |
+| `scene_ids`          | **list[str]**<br/>The scene IDs. If used, all other parameters except `geometry` are ignored.                                                                |
+| `limit_per_job`      | **int**<br/>The maximum number of expected results per job. The default value is `1`.                                                                        |
+| `geometry_operation` | **str**<br/>The geometric filter. The allowed values:<br/><ul><li>`bbox`</li><li>`intersects`</li><li>`contains`</li></ul>The default value is `intersects`. |
+
+<h5> Example </h5>
+
+```python
+workflow.construct_parameters_parallel(
+    geometries=[up42.get_example_aoi(location="Berlin")],
+    interval_dates=[("2023-01-01", "2023-01-30"), ("2023-02-01", "2023-02-26")],
+    limit_per_job=2,
+    geometry_operation="bbox",
+)
+```
+
+### estimate_job()
+
+The `estimate_job()` returns the cost estimate for a job.
+
+```python
+estimate_job(input_parameters)
+```
+
+The returned format is `dict`.
+
+<h5> Arguments </h5>
+
+| Argument           | Overview                                                |
+| ------------------ | ------------------------------------------------------- |
+| `input_parameters` | **Union[dict, str, Path]**<br/>The job JSON parameters. |
```

Comment:
1. We need to hint that they come construct_parameters method
2. str and Path case explanation are missing